Common/src/toolboxes/C1DInterpolation.cpp

```diff
@@ -0,0 +1,190 @@
+/*!
+ * \file C1DInterpolation.cpp
+ * \brief Inlet_interpolation_functions
+ * \author Aman Baig
+ * \version 7.0.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation 
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2019, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+
+#include "../../include/toolboxes/C1DInterpolation.hpp"
+
+su2double CAkimaInterpolation::EvaluateSpline(su2double Point_Interp){
+    Point_Match = false;
+
+    for (int i=0;i<n-1;i++)
+        if(Point_Interp>=x[i] && Point_Interp<=x[i+1])
+            Point_Match = true;
+
+    if (Point_Match == false){
+        cout<<"WARNING: Extrapolating data for radius: "<<Point_Interp<<endl;
+        Point_Match = true;
+    }
+
+    int i =0, j=n-1;
```

Comment:
This is a little picky, but the lack of spacing around operators makes the code a little harder to read.. if you have the ability to auto-format in your IDE, I would recommend adding consistent spaces at least for =, <, etc. (up to you on +, -, etc.)

Reply:
I would see if I can find an auto-format tool in VS code but I will fix this one up atleast.

Reply:
You can definitely do this in VS Code if you select a style, such as google, and use the auto-format feature (shift+option+f on Mac)

Reply:
I used the C, C++ style on VS code to auto format (which is only one installed on VS code), and it changed many things drastically including the for(....) {} brackets making the first bracket go to another line. And included a lot of unwanted spacing. So I just fixed this one up manually.